### PR TITLE
[v4r2] consider VO and setup settings

### PR DESCRIPTION
**Pull request by TaykYoku**
NOTE: this PR wait [#5262](https://github.com/DIRACGrid/DIRAC/pull/5262)

With these changes, the WebApp section can contain a configuration for each setup or VO, as organized in Operations:
```
Operations
{
  myVO
  {
    WebApp
    {
      Option = myVO_bla
      Schema
      { ... }
    }
  }
  mySetup
  {
    WebApp
    {
      Option = mySetup_bla
    }
  }
}
WebApp
{
  Option = some_value
  Schema
  { ... }
}
```

BEGINRELEASENOTES
This changes consider VO and setup settings in `/WebApp` section.

CHANGE: consider VO and setup settings

ENDRELEASENOTES

### CODE REVIEW

File: Lib/WebHandler.py [hunk 1]
```diff
@@ -227,11 +227,47 @@ def getProperties(self):
...
   def getAppSettings(self, app=None):
-    return Conf.getAppSettings(app or self.__class__.__name__.replace('Handler', '')).get('Value') or {}
+    return self.getSessionData('configuration').get(app or self.__class__.__name__.replace('Handler', ''), {})
+
+  def getTheme(self):
+    """ Get theme
+
+        :return: str
+    """
+    return self.getSessionData('configuration').get("Theme", "tabs")
```
Comment (fstagni) [inline Lib/WebHandler.py]: Shouldn't it be "Configuration"?

And BTW, didn't we actually say to remove the Desktop theme?

Comment (TaykYoku) [reply Lib/WebHandler.py]: IIUC now everywhere reference to "configuration" key name.

Yes, we agree that Desktop must be removed. But, as I remember, after #427 will be solved and not in v4r2, no?

**Comment by TaykYoku** [comment]
This PR was made from a branch whose name is no different from the release, so it closed. It is my mistake I will open a new one from the branch with the correct name.

**Comment by fstagni** [comment]
Why was this merged? Also, wasn't this waiting for #5262?

**Comment by TaykYoku** [comment]
Because, name of branch 'rel-v4r2'. Its merged when I make release automaticaly, without any change. Its my fault. I will make this PR again with correct branch name.

**Pull request merged**